src/transforms/pipelines/bypass.rs

```diff
@@ -0,0 +1,72 @@
+use super::expander::ExpanderConfig;
+use crate::conditions::{not::NotConfig, AnyCondition};
+use crate::config::{DataType, ExpandType, TransformConfig, TransformContext};
+use crate::transforms::filter::FilterConfig;
+use crate::transforms::Transform;
+use indexmap::IndexMap;
+use serde::{Deserialize, Serialize};
+
+/// This transform is made to do the following trick
+///
+/// ```
+///                                 +--> filter (if condition) --> ..transforms --+
+/// event -- (expand in parallel) --+                                             +--> next pipe
+///                                 +--> filter (if !condition) ------------------+
+/// ```
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct BypassConfig {
```

Comment:
It feels like there's a better name for this but I'm not entirely sure what it is. Something like `ConditionalApplication` feels more accurate than `Bypass`.